docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -19,91 +13,21 @@ Vulnerable software can have one or more vulnerabilities (CVEs).
 For Fleet Premium users, each CVE includes its CVSS base score (reported by the [National Vulnerability Database](https://nvd.nist.gov/)), probability of exploit (reported by [FIRST](https://www.first.org/epss/)), and whether or not there is a known exploit in the wild (reported by the [Cybersecurity & Infrastructure Security Agency](https://www.cisa.gov/known-exploited-vulnerabilities-catalog)).
 
 Fleet's strategy for detecting vulnerabilities (CVEs) varies according to the host's platform and
```

Comment:
I think we can move this section to the contributors doc.

This page is about letting Fleet users know what they get. They can go to the contributor docs if they want to learn how it works.

Reply:
Moved